### PR TITLE
Add recipe for argcomplete

### PR DESCRIPTION
**Pull request by asmeurer**

### CODE REVIEW

File: recipes/argcomplete/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package:
+  name: argcomplete
+  version: "1.8.2"
+
+source:
+  fn: argcomplete-1.8.2.tar.gz
+  url: https://pypi.python.org/packages/7a/60/72d44d1a4ec99c0e8a47764a65850b08bd6e682e0e70ebab2b84632c3c56/argcomplete-1.8.2.tar.gz
+  md5: 1234c0e0db38c5181d8062177361cc4b
```
Comment (synapticarbors) [inline recipes/argcomplete/meta.yaml]: Please consider using sha256

File: recipes/argcomplete/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+test:
+  imports:
+    - argcomplete
+
+about:
+  home: https://github.com/kislyuk/argcomplete
+  license: Apache
+  summary: 'Bash tab completion for argparse'
+  license_family: APACHE
```
Comment (synapticarbors) [inline recipes/argcomplete/meta.yaml]: add:

```
license_file: LICENSE.rst
doc_url: https://argcomplete.readthedocs.io
dev_url: https://github.com/kislyuk/argcomplete
```

File: recipes/argcomplete/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - argcomplete
+
+about:
+  home: https://github.com/kislyuk/argcomplete
+  license: Apache
```
Comment (synapticarbors) [inline recipes/argcomplete/meta.yaml]: Change to `Apache 2.0`

File: recipes/argcomplete/meta.yaml (hunk 2 above)
Comment (synapticarbors) [inline recipes/argcomplete/meta.yaml]: change to `Apache`

File: recipes/argcomplete/bld.bat [hunk 4]
```diff
@@ -0,0 +1,2 @@
+"%PYTHON%" setup.py install
```
Comment (synapticarbors) [inline recipes/argcomplete/bld.bat]: Consider replacing `bld.bat` and `build.sh` with the `script` option in the `meta.yaml`.

**Comment by asmeurer** [comment]
By the way I used conda skeleton pypi (if you couldn't tell). Someone really ought to update it to be able to give conda-forge compatible recipes.

**Comment by synapticarbors** [comment]
I knew these were from `conda skeleton`, and to be honest, I'm not sure how strict the community wants to be about adhering to a particular style, but I usually suggest trying to bring recipes in line with the example recipe. 

There is a recent issue about producing conda-forge compatible recipes from `conda skeleton`:
https://github.com/conda/conda-build/issues/1703#issuecomment-275538971

**Comment by ocefpaf** [comment]
@synapticarbors review is perfect to the conda-forge style. But @asmeurer has a point that we should improve conda-skeleton to be closer to our style so we need to edit less.

However, the license version is not a bug on either our style nor `conda-skeleton`. PyPI classifiers are at fault here. They do not allow for things like `BSD 2-Clause` for example.

**Comment by asmeurer** [comment]
I don't recall if skeleton prefers the classifiers or the license free-form field for the license. I know the license free-form field is problematic because people sometimes put the whole license text in there.

File: recipes/argcomplete/bld.bat (hunk 4 above)
Comment (ocefpaf) [inline recipes/argcomplete/bld.bat]: Ping @asmeurer. I guess that only this action is holding up the merge here.

File: recipes/argcomplete/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,37 @@
+package:
+  name: argcomplete
+  version: "1.8.2"
+
+source:
+  fn: argcomplete-1.8.2.tar.gz
+  url: https://pypi.python.org/packages/7a/60/72d44d1a4ec99c0e8a47764a65850b08bd6e682e0e70ebab2b84632c3c56/argcomplete-1.8.2.tar.gz
```
Comment (ocefpaf) [inline recipes/argcomplete/meta.yaml]: Can you change that URL to `https://pypi.io/packages/source/a/argcomplete/argcomplete-{{ version }}.tar.gz` in the feedsotck? (Merging this now so we don't have to wait for Travis-CI again. The backlog is quite annoying this past week.)

**Comment by asmeurer** [comment]
Speaking of the Travis backlog, I was wondering, why does the create_feedstocks script run on Travis? It's so backlogged, it takes forever just for the feedstock to be created. Wouldn't it make more sense to run it on Circle?

**Comment by SylvainCorlay** [comment]
> Speaking of the Travis backlog, I was wondering, why does the create_feedstocks script run on Travis? It's so backlogged, it takes forever just for the feedstock to be created. Wouldn't it make more sense to run it on Circle?

Yeah. Besides, it seems that the feedstock creation builds fell into the cracks of the Travis CI outage, and were neither cancelled nor processed.

**Pull request merged**